packages/blockchain-wallet-v4/src/redux/addressLabels/sagas.js

```diff
@@ -0,0 +1,26 @@
+import { put } from 'redux-saga/effects'
+import * as actions from './actions'
+import * as Types from '../../types'
+
+export default () => {
+  const generateAddressLabels = function * ({payload}) {
+    const accounts = Types.Wallet.selectHDAccounts(payload.wallet)
+
+    let addressLabels = accounts.map((account) => {
+      const hd = accounts.get(account.index)
```

Comment:
@schnogz are you doing something like this for unusedAddresses? It'll make it snippy snappy.